src/third_party/libcxx/BUILD.gn

```diff
@@ -1,3 +1,8 @@
+# Used cmake args:
+# -DLIBCXX_INCLUDE_BENCHMARKS=OFF
```

Comment:
You should list all the non-CMAKE args here - in case their default values change in future versions.